### PR TITLE
chore: Set up release-please

### PR DESCRIPTION
**Pull request by ospencer**
Closes #56 

Should be all set now!

### CODE REVIEW

**Review by phated — CHANGES_REQUESTED** [review]
One issue

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+      - master
 
 jobs:
-  microsoft-publish:
-    name: Publish to Microsoft marketplace
+  release-please:
+    name: Create Release
+    runs-on: ubuntu-latest
+    outputs:
+      release_created: ${{ steps.release.outputs.release_created }}
```
[the remark this answers is not in the document]
Comment (ospencer) [reply .github/workflows/release.yml:12]: Ah, okay.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    branches:
+      - master
+
+jobs:
+  release-please:
+    name: Create Release
+    runs-on: ubuntu-latest
+    outputs:
+      # Remap this name because it is quite confusing
+      releases_created: ${{ steps.release.outputs.releases_created }}
```
Comment (phated) [inline .github/workflows/release.yml:13]: I'm a dummy
```suggestion
      release_created: ${{ steps.release.outputs.releases_created }}
```

Comment (phated) [reply .github/workflows/release.yml:13]: Now that will match what is required in the publish step.

**Review by phated — APPROVED** [review]
I _think_ this is ready to go now 👍

**Comment by ospencer** [comment]
Let's goooooooooo!

**Pull request merged**